app/views/content/life-as-a-teacher/pay-and-benefits/_what-teachers-have-to-say.html.erb

```diff
@@ -8,22 +8,19 @@
             <ul>
               <%= render Content::InspirationCardComponent.new(heading_tag: "h3", card:
```

Comment:
@jenhadfield-dfe @emmaachesongray I think there was a conversation in UCD yesterday about whether this was too much pink in close proximity? I'm happy to take the UCD collective steer on this

![image](https://github.com/user-attachments/assets/355db674-e866-404f-8a0b-061c8590a7bf)

Reply:
Yep, we did discuss this in UCD. The feelings weren't too strong. I had a chat with Jen earlier and we agreed to leave it as-is as it aligns more closely with the branding guidelines, and then see if there's any feedback in future research sessions.